salt/modules/kapacitor.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+'''
+Kapacitor execution module.
+
```

Comment:
Ah, that's my misunderstanding of what kapacitor is. In that case, the `depends` block isn't needed.